src/include/stir/recon_buildblock/BinNormalisationFromAttenuationImage.h

```diff
@@ -92,6 +92,7 @@ class BinNormalisationFromAttenuationImage :
     This means \c multiply with the data in the projdata object 
     passed in the constructor. 
   */
+  using base_type::apply;
```

Comment:
move above the doxygen comment, as otherwise that will be applied to the wrong statement

Reply:
done